docs/limitations.rst

```diff
@@ -0,0 +1,33 @@
+.. _limitations:
+
+===================
+Runtime limitations
+===================
+
+- **Currently in private beta**
+   
+   The Qiskit runtime is currently in private beta testing with members of the
+   IBM Quantum Network.  A general release will follow after this testing is
+   completed. 
+
+- **Limited quantum systems available for qiskit runtime**
```

Comment:
```suggestion
- **Limited quantum systems available for Qiskit runtime**
```

Reply:
let me get it

Reply:
In here: https://github.com/Qiskit-Partners/qiskit-runtime/pull/22/commits/ebdd713c6a453ea3c4a8d6310b57793a29fae046